lib/departure_comparator.rb

```diff
@@ -64,12 +79,19 @@ def report_feed_down
     message
   end
 
-  def report_missing_route(route_number, headsign, stop_name, gtfs_time)
-    @messages << <<~message
+  def report_missing_route(route_number, headsign, stop_name,
+                           gtfs_time, other_headsigns)
+    message = <<~message
```

Comment:
Seems a little confusing using a heredoc signifier that is the same as the variable name. At the very least, isn't the convention to use all-caps?